### PR TITLE
Update requirements and remove scaffold files

### PR DESCRIPTION
**Pull request by yangashley**
Update requirements and remove scaffold files. 
Asana task: https://app.asana.com/0/1205655776549324/1206697443678565/f

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-click==7.1.2
-Flask==1.1.2
+click==8.1.7
+Flask==3.0.2
 Flask-Cors==3.0.10
-Flask-Migrate==2.6.0
-Flask-SQLAlchemy==2.4.4
+Flask-Migrate==4.0.5
+Flask-SQLAlchemy==3.1.1
 gunicorn==20.1.0
```
Comment (kelsey-steven-ada) [inline requirements.txt:11]: There is a newer versions of gunicorn (21.2.0) that I've updated hello-books to without issues on deployment, I think we should go with the newer version here too. Do you know if this was updated on the Flasky project or is it using 20.1.0 as well?

**Review by kelsey-steven-ada — COMMENTED** [review]
If we leave the __init__.py with create_app, it makes sense to leave the conftest with some content, but I was thinking we should remove the empty test_routes.py file and let students set that up and decide how they want to name it, like we're doing with the other removed files.

File: requirements.txt (hunk 1 above)
Comment (yangashley) [reply requirements.txt:11]: Let me check on Flasky and make the update here for gunicorn. Thanks!

**Review by kelsey-steven-ada — APPROVED** [review]
Looks great, thanks for making these updates!

**Pull request merged**